lib/matplotlib/widgets.py

```diff
@@ -2238,6 +2238,10 @@ def on_select(min: float, max: float) -> Any
         If `True`, the event triggered outside the span selector will be
         ignored.
 
+    snap_values : 1D array-like, default: None
```

Comment:
```suggestion
    snap_values : 1D array-like, optional
```
We write it this way if `None` is only the placeholder for "not given" (c.f. https://matplotlib.org/devdocs/devel/documenting_mpl.html#default-values)